platform/mellanox/fw.mk

```diff
@@ -21,7 +21,7 @@ MLNX_FW_BASE_PATH = $(MLNX_SDK_BASE_PATH)
 # Place an URL here to FW if you want to download FW instead
 MLNX_FW_BASE_URL =
 
-SIMX_VERSION = 23.7-1086
+SIMX_VERSION = 23.10-1025
```

Comment:
SIMX version shouldn't be changed, SONiC is still using 23.7-1086 regardless of SDK working with 23.10-1025.